pluginClientSdkLib/build.gradle

```diff
@@ -57,6 +58,14 @@ android.libraryVariants.all { variant ->
     }
 }
 
+if (JavaVersion.current().isJava8Compatible()) {
```

Comment:
What problem were you running into that needed this option to be set?  When building locally, I don't see any warnings with this block removed.